mev_inspect/arbitrages.py

```diff
@@ -116,13 +129,15 @@ def _get_shortest_route(
                 next_swap,
                 end_swaps,
                 other_swaps,
+                max_route_length=max_remaining_route_length,
```

Comment:
The earlier suggestions would make this ```shortest_route_length = current_shortest_route_length```, much more representative of what's going on: only search for routes below the upper limit: the current shortest route